amazon-ecs-fargate-consul-connect-example-cdk-construct/lib/amazon-ecs-fargate-consul-connect-example-cdk-construct-stack.ts

```diff
@@ -0,0 +1,509 @@
+import * as cdk from '@aws-cdk/core';
+import * as ecr from '@aws-cdk/aws-ecr';
+import * as ecs from '@aws-cdk/aws-ecs';
+import { DockerImageAsset } from '@aws-cdk/aws-ecr-assets';
+import * as ecrdeploy from 'cdk-ecr-deployment';
+import * as awsLogs from '@aws-cdk/aws-logs';
+import * as ec2 from '@aws-cdk/aws-ec2'; 
+import * as iam from '@aws-cdk/aws-iam';
+
+export class AmazonEcsFargateConsulConnectExampleCdkConstructStack extends cdk.Stack {
+  constructor(scope: cdk.App, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+//Import VPC where your consul connect server lives
+const vpc = ec2.Vpc.fromVpcAttributes(this, 'Consul-Connect-VPC', {
+  vpcId: '$CONSUL-CONNECT-VPC',
+  availabilityZones: ['$CONSUL-CONNECT-AZ'],
+  privateSubnetIds: ['$PRIVATE-SUBNETID'],
+  publicSubnetIds: ['$PUBLIC-SUBNETID']
+});
+
+//Import security group attached to the above VPC
+const importedSecurityGroup = ec2.SecurityGroup.fromSecurityGroupId(
```

Comment:
is there a way to reference the security group we need directly from the VPC we're importing on line 14, instead of having to specify it manually by ID?

also: suggest making this variable name more descriptive - "importedSecurityGroup" does not tell me what it belongs to or what it's used for